src/System.Management.Automation/CoreCLR/CorePsStub.cs

```diff
@@ -1207,20 +1207,20 @@ public RegistryValueKind GetValueKind(string name)
 
         public void Close()
         {
-            throw new NotImplementedException("Close() is not implemented. TransactedRegistry related APIs should not be hitten.");
+            throw new NotImplementedException("Close() is not implemented. TransactedRegistry related APIs should not be hidden.");
```

Comment:
Normally that would be spelled `hit`. Or one could use `run` or `executed` for clarity.